recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - ruby
     - rb-gemoji >=3.0,<4
     - rb-html-pipeline >=2.2,<3
-    - rb-jekyll >=3.0,<4
+    - rb-jekyll >=3.0,<5
```

Comment:
I guess more correct would be to also use `>=4.0`?

Reply:
It is not what they have in [rubygems](https://rubygems.org/gems/jemoji)